deploy/docker/Dockerfile

```diff
@@ -1,28 +1,44 @@
 FROM alpine:3.5
 
-LABEL maintainer="adundovi@gmail.com"
-LABEL description="Prepares a docker image \
-of pump.io server, http://pump.io/."
+LABEL maintainer Andrej Dundovic <adundovi@gmail.com>
+LABEL description "Stream server that does \
+most of what people really want from a social network."
 
 ENV LC_ALL en_US.UTF-8
 
+ENV PUMPIO_VERSION 3.0.2
+
 RUN apk add --no-cache \
-  nodejs \
-  git \
-  python \
-  make \
-  g++ \
-  graphicsmagick
-RUN npm install -g databank-mongodb
-RUN npm install -g pump.io
-RUN adduser -D -g "" pumpio
+    curl \
+    nodejs \
+    python \
+    make \
+    g++ \
+    git \
+    graphicsmagick \
+ && adduser -D -g "" pumpio \
+ && mkdir -p /etc/ssl/private \
+ && npm install -g databank-mongodb@0.19.2 \
+ && mkdir -p /usr/src \
+ && curl -fSL https://github.com/pump-io/pump.io/archive/v$PUMPIO_VERSION.zip -o pumpio.zip \
```

Comment:
Hmm, but this part I don't understand. If I would develop something, I would never do it through a docker image because it would slow down my workflow: rebuilding the image every time when I want to test something doesn't make sense. Docker is primary here for deployment. One can do the development through some other techniques, like LXC. A prescription how to set up the developing environment is already here, in this Dockerfile.

Also, it would make sense to decouple the docker image creation from the original source, ideally to have a separate repository for it which means that the local copy is not available anymore.

Building a docker image shouldn't depend on something outside of the folder where the Dockerimage file is located (it is against the rules). Either we would put this Dockerimage inside the root folder of this repository (../..), or we keep it like this. A script can be made which would update the dockerfile once you publish a new version with publish, right?

Generally, I don't agree with this comment, but it seems that the disagreement arises due to different visions for what Docker will be used. We should clarify it. For me it is used for the stable deployment of Pump.io and for you it some sort of development env, right?